### PR TITLE
[FrameworkBundle] Added doc for ControllerTrait::isFormValid

### PR DESCRIPTION
**Pull request by lyrixx**
refs: https://github.com/symfony/symfony/pull/24576

### CODE REVIEW

File: best_practices/forms.rst [hunk 1]
```diff
@@ -218,4 +218,22 @@ handling the form submit. For example, you *could* have a ``new()`` action that
 submit. Both those actions will be almost identical. So it's much simpler to let
 ``new()`` handle everything.
 
+.. versionadded:: 4.3
+   The ``ControllerTrait::isFormValid()`` method was added in Symfony 4.3.
+
+It's also possible to use a short-cut if the ``ControllerTrait`` is imported:
```
Comment (nicolas-grekas) [inline best_practices/forms.rst]: the trait is internal, we should tell about AbstractController instead.

File: best_practices/forms.rst [hunk 2]
```diff
@@ -218,4 +218,22 @@ handling the form submit. For example, you *could* have a ``new()`` action that
...
 ``new()`` handle everything.
 
+.. versionadded:: 4.3
+   The ``AbstractController::isFormValid()`` method was added in Symfony 4.3.
+
+It's also possible to use a short-cut if the ``AbstractController`` is extended:
+
+.. code-block:: php
+
+    public function newAction(Request $request)
```
Comment (gregurco) [inline best_practices/forms.rst]: I think it's better to follow best practices:
> Don't add the Action suffix to the methods of the controller actions.

Link: https://symfony.com/doc/current/best_practices/controllers.html#controller-action-naming

**Comment by lyrixx** [comment]
@OskarStark Thanks for the review. I updated the PR

File: best_practices/forms.rst (hunk 2 above)
Comment (lyrixx) [reply best_practices/forms.rst]: You are totally right. I never add it in my own code :joy: I don't know why I did it there. 
Anyway, fixed

File: best_practices/forms.rst [hunk 3]
```diff
@@ -218,4 +218,21 @@ handling the form submit. For example, you *could* have a ``new()`` action that
 submit. Both those actions will be almost identical. So it's much simpler to let
 ``new()`` handle everything.
 
+.. versionadded:: 4.3
+
+   The ``AbstractController::isFormValid()`` method was added in Symfony 4.3.
+
+It's also possible to use a short-cut if the ``AbstractController`` is extended::
```
Comment (nicolas-grekas) [inline best_practices/forms.rst]: [...] if your controller extends AbstractController:

File: forms.rst [hunk 4]
```diff
@@ -264,6 +264,25 @@ your controller::
     is called. Otherwise, changes done in the ``*_SUBMIT`` events aren't applied to the
     view (like validation errors).
 
+
+.. versionadded:: 4.3
+
+   The ``AbstractController::isFormValid()`` method was added in Symfony 4.3.
+
+It's also possible to use a short-cut if the ``AbstractController`` is extended::
```
Comment (nicolas-grekas) [inline forms.rst]: [...] if your controller extends AbstractController:

File: best_practices/forms.rst (hunk 3 above)
Comment (lyrixx) [reply best_practices/forms.rst]: done. Thanks

**Review by gregurco — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Pull request closed without merging**